azure-pipelines.yml

```diff
@@ -102,6 +102,40 @@ jobs:
         ./scripts/ci/verify_modified_index.sh
       displayName: "CLI Linter on Modified Extension"
 
+- job: LintModifiedExtensionsNew
```

Comment:
The word `New` is quite tricky...

Reply:
I will rename it to the old one once approved. I can change it right now. But for the comparison purpose I left the old one.